app/modules/kit-library/assets/js/hooks/use-kits.js

```diff
@@ -14,7 +14,7 @@ export const KEY = 'kits';
 export const defaultQueryParams = {
 	favorite: false,
 	search: '',
-	taxonomies: taxonomyType.reduce( ( current, { key } ) => {
+	taxonomies: TAXONOMIES.reduce( ( current, { key } ) => {
```

Comment:
strange naming here

Reply:
Should I just lowercase it? Something else?